src/objects/Mesh.js

```diff
@@ -160,9 +160,9 @@ THREE.Mesh.prototype.raycast = ( function () {
 							distance: distance,
 							point: intersectionPoint,
 							face: new THREE.Face3( a, b, c, THREE.Triangle.normal( vA, vB, vC ) ),
-							faceIndex: null,
+							faceIndex: Math.floor(i/3), // triangle number in indices buffer semantics
```

Comment:
I do not use drawcalls. I hope the person who will use them, will make an enhancement. BTW current enhancement is good, even it is not consider drawcalls.
p.s. I just do not understand those drawcalls. I googled and found no comprehensive info about them.